aten/src/ATen/Declarations.cwrap

```diff
@@ -1353,7 +1352,6 @@
   types:
     - floating_point
   backends:
-    - CPU
```

Comment:
why can't you kill the entire entry instead of just the CPU one?  How are these called?

Reply:
They're called from the stubs in CUDAUnaryOps.cpp

Reply:
Actually turns out they're not so I'm gonna delete these